docs/DeveloperGuide.md

```diff
@@ -921,11 +929,51 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   i. Download the jar file and copy into an empty folder
 
-   2. Double-click the jar file Expected: Shows the GUI with a set of sample orders and items. 
+   ii. Double-click the jar file Expected: Shows the GUI with a set of sample orders and items. 
    The window automatically expands to occupy the entire screen.
 
+### Listing all orders
+
+1. Listing all orders after calling `findo`
+  * Prerequisites: List all orders using the `listo` command. There are already multiple existing orders.
+    Of these orders, there exist 2 orders made by customers with `Chan` in their names. Before testing, enter `findo n/Chan`<br/>
+    Expected: Order list displays 2 orders by customers with `Chan` in their name.
+  * Test case: Enter `listo` into the command box.<br/>
+    Expected: ALl orders listed in the order list.
+  * Alternative negative test case: Enter `listO` into the command box.<br/>
```

Comment:
I think it would be good to standardize this part
```suggestion
  * Other incorrect `listo` commands to try: Enter `listO` into the command box.<br/>
  Expected: Similar to previous.
```